src/sqlfluff/dialects/dialect_exasol.py

```diff
@@ -2420,6 +2420,7 @@ class CreateUserSegment(BaseSegment):
             Ref("UserPasswordAuthSegment"),
             Ref("UserKerberosAuthSegment"),
             Ref("UserLDAPAuthSegment"),
+            Ref("UserOPENIDAuthSegment"),
```

Comment:
Nit but it’s OpenID. Should we replace all instances (except the keyword itself) with that capitalisation?

Reply:
done